nav2_amcl/src/amcl_node.cpp

```diff
@@ -1106,6 +1111,7 @@ AmclNode::initParameters()
   get_parameter("always_reset_initial_pose", always_reset_initial_pose_);
   get_parameter("scan_topic", scan_topic_);
   get_parameter("map_topic", map_topic_);
+  get_parameter("freespace_downsampling", freespace_downsampling_);
```

Comment:
Perhaps set this as a configuratable downsampling step? i.e. 2,3,4,5,... might be nice to experiment with yourself.

Reply:
We have a comically large map and 1/th is already enough to make us happy